spec/factories/users.rb

```diff
@@ -0,0 +1,16 @@
+FactoryBot.define do
+  factory :user do
+    id { 1 }
+    name { 'sample' }
+    email { 'sample@example.com' }
+    password { '00000000' }
+    admin { false }
+  end
+  factory :admin_user, class: User do
+    id { 2 }
+    name { 'admin' }
+    email { 'admin@example.com' }
+    password { '11111111' }
```

Comment:
テストデータのパスワードは同じ値で統一した方が、開発上都合が良いかと思います。
また、数値型を文字列に変換してしてする必要があるか、確認をお願いします。
```suggestion
    password { 00000000 }
```